trl/trainer/callbacks.py

```diff
@@ -234,11 +234,15 @@ def __init__(
         trainer: Trainer,
         generation_config: Optional[GenerationConfig] = None,
         num_prompts: int = None,
+        shuffle_order: Optional[bool] = True,
```

Comment:
This was never exposed, so the user couldn't change it

Reply:
Nice, can you document it?